src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1742,12 +1742,12 @@ public String toString(Attribute a, int flags) {
 					
 					@Override
 					public String toVariableNameString(Attribute a) {
-						return a.name();
+						return toString(a, 0);
 					}
 					
 					@Override
 					public String getVariableNamePattern() {
-						return "\\S+";
+						return "[\sA-Za-z]+";
```

Comment:
annoying thing about java regex: gotta escape the `\`, so `\s` would be `\\s`

Reply:
Was fixed previously in https://github.com/SkriptLang/Skript/pull/2664/commits/9068b60f4f90cd44272878e29a2a8a51e0594e8c